.buildkite/auditbeat/auditbeat-pipeline.yml

```diff
@@ -19,50 +19,140 @@ env:
   AWS_ARM_INSTANCE_TYPE: "t4g.xlarge"
 
 steps:
-  - input: "Input Parameters"
```

Comment:
as discussed let's not remove the input parameters, unless we already know that this is not needed.

Reply:
We need to remove it fro these pipelines anyway

Reply:
How would it work if someone needs to trigger a job from the UI then?

Reply:
As discussed over Zoom, it would make sense to understand if we need to keep those or not. 
The current Jenkins has this capability and since we are migrating we should not change the functionality.
When we agree that this is something we don't need then it would make sense to remove it. But given that this can take some time to understand I am not sure how possible it is to do now.

Reply:
I'm touching base again on this. I was looking at https://github.com/elastic/ingest-dev/issues/3050, and it seems like we can indeed remove the UI parameters for now.

Reply:
Awesome! 👍